statscache/app.py

```diff
@@ -36,24 +28,36 @@ def main(name):
     )
 
 
+@app.route('/')
+def index():
+    """ Generate a JSON-P response with an index of plugins (as an array) """
+    return jsonp(json.dumps(plugins.keys()), 200)
```

Comment:
Awesome! I'll wait 'till tomorrow to do anything with this, so that @rtnpro has a chance to chime in.